src/components/ActionInput/ActionInput.vue

```diff
@@ -307,8 +307,8 @@ $input-margin: 4px;
 		align-items: center;
 		justify-content: center;
 
-		width: #{$clickable-area - $input-margin * 2};
-		height: #{$clickable-area - $input-margin * 2};
+		width: #{$clickable-area - $input-margin * 6};
+		height: #{$clickable-area - $input-margin * 6};
```

Comment:
But it doesn't make any sense to have 6 here.
The 2 were here to compensate from height/width.

Reply:
`box-sizing: border-box;` might be the better option here.

Reply:
Works fine with `border-box` indeed.

Reply:
Iwonder if that wouldn't make sense to apply this to the whole actions component with the `*` selector. I thought I saw a comment about this somewhere, but can't find it.

If we want to be as independent from server's styling as possible, this might be better :thinking:

Reply:
cc @raimund-schluessler

Reply:
> Iwonder if that wouldn't make sense to apply this to the whole actions component with the `*` selector. I thought I saw a comment about this somewhere, but can't find it.
> 
> If we want to be as independent from server's styling as possible, this might be better 🤔

I guess this should be ok, since we have this in the server anyway (only for content, but that effects most elements):
```
#content[class*='app-'] * {
 box-sizing:border-box
}
```

But if we do that, we could as well just do it globally for all vue components.

Reply:
> But if we do that, we could as well just do it globally for all vue components.

That is what I meant

Reply:
> 
> 
> > But if we do that, we could as well just do it globally for all vue components.
> 
> That is what I meant

Ah, ok. I thought you meant to only apply it to the Actions component. If we have it globally, we could also get rid of this https://github.com/nextcloud/nextcloud-vue/blob/master/tests/visual/components/AppSidebar/AppSidebar.visual.js#L40-L49 and this https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Content/Content.vue#L73-L75, I think.

Reply:
Sorry, my message wasn't very clear :p
what do you think @tcitworld ?